pkg/openstack/loadbalancer_test.go

```diff
@@ -593,3 +594,195 @@ func Test_getIntFromServiceAnnotation(t *testing.T) {
 		})
 	}
 }
+
+func Test_buildPoolCreateOpt(t *testing.T) {
+	type args struct {
+		protocol string
+		svcConf  *serviceConfig
+		service  *corev1.Service
+		lbaasV2  *LbaasV2
+	}
+	tests := []struct {
+		name string
+		args args
+		want pools.CreateOpts
+	}{
+		{
+			name: "test for proxy protocol enabled",
+			args: args{
+				protocol: "TCP",
+				svcConf: &serviceConfig{
+					keepClientIP:                true,
+					tlsContainerRef:             "tls-container-ref",
+					healthMonitorDelay:          1,
+					healthMonitorTimeout:        1,
+					healthMonitorMaxRetries:     1,
+					healthMonitorMaxRetriesDown: 1,
+					healthCheckNodePort:         2,
+					enableProxyProtocol:         true,
+				},
+				lbaasV2: &LbaasV2{
+					LoadBalancer{
+						opts: LoadBalancerOpts{
+							LBProvider: "ovn",
+							LBMethod:   "ROUND_ROBIN",
```

Comment:
Not really important, feel free to ignore, but this combinantion wouldn't work in real world, `ovn` only works with `SOURCE_IP_PORT`.

Reply:
noted, but. i'll update it just so that it's not misleading to people reading it in the future